backend/src/CronJob.js

```diff
@@ -0,0 +1,341 @@
+import TransactionService from "./services/TransactionService";
+import PocketService from "./services/PocketService";
+import EmailService from "./services/EmailService";
+import NodeService from "./services/NodeService";
+import CronJobService from "./services/CronJobService";
+import ApplicationService from "./services/ApplicationService";
+import cron from "node-cron";
+
+import {RpcError, typeGuard} from "@pokt-network/pocket-js";
+import {POST_ACTION_TYPE} from "./models/Transaction";
+import {BOND_STATUS, STAKE_STATUS} from "./models/CronJobData";
+
+export function startCronJobs() {
+    const TRANSACTION_SERVICE = new TransactionService();
+    const POCKET_SERVICE = new PocketService();
+    const APPLICATION_SERVICE = new ApplicationService();
+    const NODE_SERVICE = new NodeService();
+    const CRON_SERVICE = new CronJobService();
+
+    cron.schedule("0 */20 * * * *", async () => {
+        let height = await POCKET_SERVICE.getBlockHeight();
+        const cronJobData = await CRON_SERVICE.getEntity();
+
+        height = Number(height);
+
+        if (height !== cronJobData.lastHeight) {
+            runPendingTransaction(cronJobData);
+            runNodeStakeProcess(cronJobData);
+            runNodeUnstakeProcess(cronJobData);
+            runAppStakeProcess(cronJobData);
+            runAppUnstakeProcess(cronJobData);
+
+            runNodeUnjailProcess(cronJobData);
+
+            cronJobData.lastHeight = height;
+            CRON_SERVICE.update(cronJobData.id, cronJobData);
+        }
+    });
+
+    function runPendingTransaction(cronJobData) {
+        const start = async () => {
+            await asyncForEach(cronJobData.pendingTransactions, async (pocketTransaction) => {
+                try {
+                    const hash = pocketTransaction.hash;
+                    const transactionOrError = await POCKET_SERVICE.getTransaction(hash);
+
+                    if (typeGuard(transactionOrError, RpcError)) {
+                        throw new Error(transactionOrError.message);
+                    }
+
+                    const transaction = transactionOrError;
+
+                    if (transaction.hash === hash) {
```

Comment:
Add a ToDo note here, since we should change this check in the future after pocket-core perform some changes that will allow us to check if it was a success or failed